nodes/src/nodes/eval_cobalt/IGlobal.py

```
@@ -0,0 +1,96 @@
+            # Validate threshold is within bounds
+            threshold = config.get('threshold', 0.7)
+            try:
+                threshold = float(threshold)
+                if threshold < 0.0 or threshold > 1.0:
+                    warning('Threshold must be between 0.0 and 1.0')
+                    return
+            except (ValueError, TypeError):
+                warning('Threshold must be a valid number between 0.0 and 1.0')
+                return
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)